### DISCUSSION TITLE
Windows 10 cannot detect the header file of Linux in WSL after updating 1.8.0 or 1.8.1

### DISCUSSION THREAD

**Discussion by xiaoyongcheng** [microsoft/vscode-cpptools#8767] [Q&A] [answered]
I don't know if it's a bug, but this problem occurred after the automatic update one day. It was solved by returning version 1.7.1.  need I to add all the header file paths to the includepath？
I think the following fixes caused it (but not sure):
Fix GCC system include processing on Windows. [#8112](https://github.com/microsoft/vscode-cpptools/issues/8112), [#8496](https://github.com/microsoft/vscode-cpptools/issues/8496)

hope you can answer the question. thx Orz

**Comment by sean-mcmanus** [comment, marked as answer]
We don't support accessing Linux/WSL paths from Windows...we had some legacy support, but we're not maintaining it. We intend for access to Linux/WSL files to be done via using the Remote WSL extension and running our extension from there (running in WSL).

**Reply by xiaoyongcheng** [reply]
It's solved. Thank you

Open with code command in WSL
There was a problem when I tried to use the code command before. It may be an extended version problem
thanks again.
